models/classes/class.ItemCompiler.php

```diff
@@ -27,16 +27,8 @@
  * @package taoItems
  
  */
-class taoItems_models_classes_ItemCompiler extends tao_models_classes_Compiler
+abstract class taoItems_models_classes_ItemCompiler extends tao_models_classes_Compiler
 {
-    /**
-     * @throws common_exception_BadRequest
```

Comment:
if you'd put @inheritdoc here nothing would change

Reply:
But changing the class to abstract is a cleaner solution than keeping it as a concrete class with a function that needs to be overridden to make it usable.